src/cmd-buildextend-live

```diff
@@ -317,6 +317,12 @@ def generate_iso():
     kargs = ' '.join(kargs_array)
     print(f'Substituting ISO kernel arguments: {kargs}')
 
+    files_with_karg_embed_areas = {}
+    cmdline = ''
+    cmdline_filename = '.cmdline'
+    files_with_karg_embed_areas[cmdline_filename] = 0
```

Comment:
Hmm, is this just relying on the iteration later on (where we write to the header) yielding this key first? I think we want something stronger here. E.g. let's keep this in a separate variable, and make sure it gets added first to the `offsets` variable, i.e. at `offsets[0]`? And also let's describe in the spec that the first offset is for the defaults just to have it documented.

Reply:
done